tests/rptest/services/redpanda.py

```diff
@@ -4121,7 +4145,36 @@ def _set_trace_loggers_and_sleep(self, node, time_sec=10):
         except Exception as e:
             self.logger.warn(f"Error setting trace loggers: {e}")
 
+    def _update_usage_stats(self, node):
+        self.logger.info(">>> updating stats: %s", node.account.hostname)
+
+        def _metrics_sum(name: str) -> int:
+            try:
+                samples = self.metrics_sample(name, [node])
+
+                if samples is None:
+                    return 0
+
+                return sum(s.value for s in samples.samples)
+            except Exception as e:
+                self.logger.warning(f"Cannot check metrics - {e}")
+                return 0
+
+        self._usage_stats.bytes_read += _metrics_sum(
+            "vectorized_io_queue_total_read_bytes_total")
+
+        self._usage_stats.bytes_written += _metrics_sum(
+            "vectorized_io_queue_total_write_bytes_total")
+        self._usage_stats.batches_read += _metrics_sum(
+            "vectorized_storage_log_batches_read")
+
+        self._usage_stats.batches_written += _metrics_sum(
+            "vectorized_storage_log_batches_written")
+
     def stop_node(self, node, timeout=None, forced=False):
+        # collect usage stats before the node is stopped, the usage stats
+        # accumulate metrics from all the nodes before they are stopped.
+        self._update_usage_stats(node)
```

Comment:
What happens if a node is stopped, restarted and then stupped again? If prometheus metrics are not zeroed they will be counted twice.

Reply:
Metrics are zeroed every time the node restarts